modules/packages/Curl.chpl

```diff
@@ -395,8 +396,9 @@ module Curl {
 
     import Sys;
     import Time;
-    private use IO;
+    use IO;
     use Curl;
+    use Sys;
```

Comment:
It looks like we have import Sys followed by use Sys; maybe just use Sys ?

Reply:
Good catch, I hadn't noticed that, thanks.